thoth/storages/graph/postgres.py

```diff
@@ -5701,6 +5701,15 @@ def cache_clear(self) -> None:
         """Drop cache of records."""
         for method in self._CACHED_METHODS:
             method.cache_clear()
+            
+    def get_database_size(self) -> int:
+        """Get size of the database in MiB."""
+        query = f"SELECT pg_database_size(pg_database.datname) AS size_in_mb FROM "\
+                f"pg_database WHERE pg_database.datname='{self._engine.url.database}'"
+
+        with self._session_scope() as session:
+            result = session.execute(query).fetchone()
+            return result[0]
```

Comment:
is it returning MB?

```
from thoth.storages import GraphDatabase
graph = GraphDatabase()
graph.connect()
graph.get_database_size()

17048657031
```

Reply:
In bytes, adjusted and rebased in the most recent commit. PTAL.

Reply:
Thanks!!